src/main/java/tennisclub/entity/User.java

```diff
@@ -29,61 +31,67 @@
     @Enumerated
     private Role role;
 
+    @ManyToMany(mappedBy = "users")
+    private Set<Booking> bookings = new HashSet<>();
+
+    @ManyToMany(mappedBy = "teachers")
+    private Set<Lesson>  lessonsToTeach = new HashSet<>();
+
+    @ManyToMany(mappedBy = "students")
+    private Set<Lesson> lessonsToAttend = new HashSet<>();
+
+    @OneToMany(mappedBy = "user")
+    private Set<Ranking> rankings = new HashSet<>();
```

Comment:
Shouldn't be ranking the owning side? 👀 I might be mistaken though 🤔

Reply:
The side that doesn't contain the `mappedBy` is the owning side. So in this case it is indeed Ranking.

Reply:
Yeah, it is slightly confusing as `user` is in this case an attribute of Ranking. However thanks to this I noticed I changed the name of the attribute from `user` to `player` and forgot to change it here :)